timeseries/src/autogluon/timeseries/models/gluonts/__init__.py

```diff
@@ -8,10 +8,12 @@
     SimpleFeedForwardModel,
     TransformerModel,
 )
+from .pytorch.models import DeepARPyTorchModel
 
 __all__ = [
     "AutoTabularModel",
     "DeepARModel",
+    "DeepARPyTorchModel",
```

Comment:
Probably in the presets we eventually want `DeepAR` to correspond to the PyTorch version, and the MXNet models should be renamed to something like `MxDeepAR`?

Reply:
Thanks!

I'm rather for changing `AbstractGluonTSPytorchModel` to `PyTorch`. this is more consistent with GluonTS's own conventions and our own conventions where we usually keep caps (`AbstractGluon*TS*Model`)

For MxDeepAR I completely agree, but could not decide whether we should just do this in the aliases in presets or also in the class names. Class names are not in the public API, so maybe it should be fine to change altogether. What do you think?

Reply:
I think renaming the classes is the way to go since our [documentation of the model zoo](https://auto.gluon.ai/dev/tutorials/timeseries/forecasting-model-zoo.html) relies on the class names, not aliases in presets.